rectools/visuals/__init__.py

```diff
@@ -22,11 +22,13 @@
 ---------
 `visuals.VisualApp` - Jupyter app for visual comparison of recommendations
 `visuals.ItemToItemVisualApp` - Jupyter app for visual comparison of item-to-item recommendations
+`visuals.MetricsApp` - Jupyter app for visual metrics comparison
 """
 
 try:
+    from .metrics_app import MetricsApp
```

Comment:
It is possible that user installed ipywidgets but didn't install plotly. In this case we need to give him full version of VisualApp, not dummy one from `compat`. And a dummy version of MetricsApp.
So we need to have separate try/except statements for imports of these classes

Reply:
Done